conf/routes

```diff
@@ -96,6 +96,7 @@ GET    /v3/participants/:userId/requestInfo                            @org.sage
 DELETE /v3/participants/:userId/activities                             @org.sagebionetworks.bridge.play.controllers.ParticipantController.deleteActivities(userId: String)
 GET    /v3/participants/:userId/activities/:activityType/:referentGuid @org.sagebionetworks.bridge.play.controllers.ParticipantController.getActivityHistoryV3(userId: String, activityType: String, referentGuid: String, scheduledOnStart: String ?= null, scheduledOnEnd: String ?= null, offsetKey: String ?= null, pageSize: String ?= null)
 GET    /v3/participants/:userId/activities/:activityGuid               @org.sagebionetworks.bridge.play.controllers.ParticipantController.getActivityHistoryV2(userId: String, activityGuid: String, scheduledOnStart: String ?= null, scheduledOnEnd: String ?= null, offsetBy: String ?= null, offsetKey: String ?= null, pageSize: String ?= null)
+POST   /v3/participants/:userId/createSmsNotificationRegistration      @org.sagebionetworks.bridge.play.controllers.ParticipantController.createSmsNotificationRegistration(userId: String)
 GET    /v3/participants/:userId/notifications                          @org.sagebionetworks.bridge.play.controllers.ParticipantController.getNotificationRegistrations(userId: String)
```

Comment:
POST /v3/participants/:userId/notifications would be more consistent

Reply:
As per IM converstion, we are renaming this to POST /v3/participants/:userId/notifications/sms and we're moving this to an internal-only API.